caffe2/CMakeLists.txt

```
@@ -1781,7 +1781,7 @@ if(BUILD_TEST)
   foreach(test_src ${Caffe2_CPU_TEST_SRCS})
     get_filename_component(test_name ${test_src} NAME_WE)
     add_executable(${test_name} "${test_src}")
-    target_link_libraries(${test_name} torch_library gtest_main)
+    target_link_libraries(${test_name} torch_library gtest_main stdc++)
```

At the very least you need to do this, as stdc++ is not available on Windows
```suggestion
if(not MSVC)
    target_link_libraries(${test_name} torch_library gtest_main stdc++)
endif()
```

---

No, we should use 
```
 target_link_libraries(${test_name} torch_library gtest_main  )
if(NOT MSVC)
    target_link_libraries(${test_name}  stdc++)
endif()
```